include/wil/cppwinrt_register_com_server.h

```diff
@@ -0,0 +1,107 @@
+//*********************************************************
+//
+//    Copyright (c) Microsoft. All rights reserved.
+//    This code is licensed under the MIT License.
+//    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF
+//    ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+//    TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+//    PARTICULAR PURPOSE AND NONINFRINGEMENT.
+//
+//*********************************************************
+//! @file
+//! Utilities for making managing COM server easier
+
+#ifndef __WIL_CPPWINRT_REGISTER_COM_SERVER_DEFINED
+#define __WIL_CPPWINRT_REGISTER_COM_SERVER_DEFINED

```

Comment:
```suggestion
#ifndef __WIL_CPPWINRT_REGISTER_COM_SERVER_INCLUDED
#define __WIL_CPPWINRT_REGISTER_COM_SERVER_INCLUDED
```
Just to match the other files